databook/background/experimental-setup.md

```diff
@@ -0,0 +1,58 @@
+# Behavioral apparatus
+
+Neurons in the visual system often have spatially selective receptive fields,
+that is, they tend to responds to stimuli shown in a particular part of visual
+space. In order to repeatably show stimuli to a mouse, the mice need to be
+head-fixed. For all the datasets described here, this means that mice have had a
+head bar that was surgically implanted prior to the experiment. For more details
+about head bar see {cite:t}`groblewski2020`.
+
+:::{figure} ../resources/Groblewski_head_fixed_mouse.jpg
+---
+width: 800
+name: mouse-with-headbar
+---
+A mouse with the head bar. Image from {cite:t}`groblewski2020`.
+:::
+
+This head-bar can clamped into a recording setup and go back to nearly the same
+location for each clamp cycle. The clamp used in these experiments does so with
+< 10um precision.
```

Comment:
This level of precision is particularly important in Ca2+ imaging experiments, because it allows the experimenter to reliably return to the same cells across multiple sessions. Further, in experiments that include ISI, it allows for targeting recordings to specified structures.